tests/test_configurations/test_base.py

```diff
@@ -1138,25 +1139,26 @@ class TestConfigurationContainingPathSerialization:
     def setup(self):
         """Setup test"""
 
+        # starting from posix-compliant format
         self.raw_paths = [
-            r"C:\Documents\Newsletters\Summer2018.pdf",
-            r"\Program Files\Custom Utilities\StringFinder.exe",
-            r"2018\January.xlsx",
-            r"..\Publications\TravelBrochure.pdf",
-            r"C:\Projects\apilibrary\apilibrary.sln",
-            r"C:Projects\apilibrary\apilibrary.sln",
-            r"c:\temp\test-file.txt",
-            r"\\127.0.0.1\c$\temp\test-file.txt",
-            r"\\LOCALHOST\c$\temp\test-file.txt",
-            r"\\.\c:\temp\test-file.txt",
-            r"\\?\c:\temp\test-file.txt",
-            r"\\.\UNC\LOCALHOST\c$\temp\test-file.txt",
-            r"jquery-1.1.1.js",
-            r"jquery-1.1.1.min.js",
-            r"jquery-1.1.1-vsdoc.js",
-            r"jquery-1.2.1-vsdoc.js",
-            r"jquery-1.2.1.js",
-            r"jquery-1.2.1.min.js",
+            "C:/Documents/Newsletters/Summer2018.pdf",
+            "/Program Files/Custom Utilities/StringFinder.exe",
+            "2018/January.xlsx",
+            "../Publications/TravelBrochure.pdf",
+            "C:/Projects/apilibrary/apilibrary.sln",
+            "C:Projects/apilibrary/apilibrary.sln",
+            "c:/temp/test-file.txt",
+            "//127.0.0.1/c$/temp/test-file.txt",
+            "//LOCALHOST/c$/temp/test-file.txt",
+            "//c:/temp/test-file.txt",
+            "//?/c:/temp/test-file.txt",
+            "//UNC/LOCALHOST/c$/temp/test-file.txt",
+            "jquery-1.1.1.js",
+            "jquery-1.1.1.min.js",
+            "jquery-1.1.1-vsdoc.js",
+            "jquery-1.2.1-vsdoc.js",
+            "jquery-1.2.1.js",
+            "jquery-1.2.1.min.js",
```

Comment:
There is some implicit conversion happening in python depending on your OS. We opt to start with POSIX-compliant path format because of this. 
```
>>> r"C:\Documents\Newsletters\Summer2018.pdf"
'C:\\Documents\\Newsletters\\Summer2018.pdf'
>>> Path(r"C:\Documents\Newsletters\Summer2018.pdf").parts
('C:\\Documents\\Newsletters\\Summer2018.pdf',)
```

opened a related issue for stripping off the root: #440